lib/client.js

```diff
@@ -79,7 +79,7 @@ client.prototype.request = function(method, path, params, callback){
 
     if(["POST", "PUT"].indexOf(http_options["method"]) >= 0){
         http_options["headers"] = {
-            "Content-Type": "application/json",
+            "Content-Type": params["contentType"] ? params["contentType"] : "application/json",
```

Comment:
hmm, wonder why they did this, since every other endpoint uses `application/json`